src/CallbackManager.jl

```diff
@@ -0,0 +1,43 @@
+#=
+    CallbackManager - TODO WIP
+
+This module facilitates calendar functions and temporal interpolations
+of data.
+=#
+
+module CallbackManager
+
+using ..Utilities
+using Dates
+
+export current_date, strdate_to_datetime, datetime_to_strdate, calendar_callback
+
+# TODO remove calls to calendar_callback - can be replaced with `<`
```

Comment:
Is this TODO for a later PR or the comment is outdated? I don't see calls to `calendar_callback` in this file

Reply:
This `TODO` is for a later PR when I go through the `CallbackManager` module in more detail. I just put the reminder here since it's where `calendar_callback` is defined.